### PR TITLE
I18n sync Down & Out 11/30

### PR DESCRIPTION
**Pull request by deploy-code-org**
# I18n Sync Down & Out

This PR contains all changes to translations made since the last sync (usually a week).

## To Review:

Look through the changes in each of the commits and verify that all changes are expected. Note that **it is not necessary to review every single "dashboard i18n update" commit.** These commits are batched by language to make it easier to review just a couple commits in this category.

### Things to Note:

- We expect translation changes to come in on a regular basis; new strings will be translated, existing translations will be updated, and source string removals will result in translations for those strings also being removed.

- We expect translations to arrive in large batches, so don't worry if you see a large group of changes in a specific language.

- We _do not_ usually expect to see a large number of formatting changes, or batches of changes that apply to strings in every single language. If you see this, it usually means either that a code change went in or that something broke.

## Known Issues:

- If the CI unit tests fail at the `grunt messages:all` step, they will typically fail with a message like:

  ```
  [2019-10-04 02:38:57] Running "messages:all" (messages) task
  [2019-10-04 02:39:00] Warning: Error processing localization file i18n/common/zh_tw.json: SyntaxError: Expected [0-9a-zA-Z$_] but "\u8A0A" found. Use --force to continue. 
  ```

  This usually means that a translator has translated one of the [MessageFormat variables](https://messageformat.github.io/messageformat/guide/#variables). In this case, you should find and fix the offending translation in crowdin, and then manually update the associated strings in `i18n/locales` and `apps/i18n`.

  See https://github.com/code-dot-org/code-dot-org/pull/31098/commits/ab3e5d1d740869a880f2e9da5cf0a4348886915e and https://github.com/code-dot-org/code-dot-org/pull/31098/commits/0fca37658cacaaff55d9adfc9d04c40b37cc5e2d for an example.

## To Deploy:

Once one or two people have reviewed and approved this change and the tests are passing, ship it!

It's not necessary to wait for everyone on the team to review.

### CODE REVIEW

File: dashboard/config/locales/blocks.tr-TR.yml [hunk 1]
```diff
@@ -13,8 +13,8 @@ tr:
             '"just right"': harfi harfine
             '"too little"': çok az
             '"too much"': çok fazla
-        text: |2-
-           if {CONDITION}{SPRITE}
+        text: |-
+          Eğer {CONDITION}{SPRITE}
```
Comment (mgc1194) [inline dashboard/config/locales/blocks.tr-TR.yml]: Not sure if that "2" needs to be in the translation or not.

File: dashboard/config/locales/resources.es-ES.json [hunk 2]
```diff
@@ -12921,7 +12921,7 @@
         },
         "generative_images_activity_guide/how-ai-works/2023": {
           "name": "[Key] Guía de actividades de imágenes generativas",
-          "url": "https://docs.google.com/document/d/1f9K9TA6gjNM5SYZB-yCovGA-agtWJdl-ATbUg9c4K_s/edit",
+          "url": "https://docs.google.com/document/d/1do-LGHpTw8VtnqMnjlcpVCqyon9IcRL8bA9i-SaTzs0/edit",
```
Comment (mgc1194) [inline dashboard/config/locales/resources.es-ES.json]: This file is in Portuguese.

**Comment by deploy-code-org** [comment]
This comment is created by running: `bin/i18n/analyze_pr.rb -p 55252 -c`

Top 5/707 files sorted by the number of changes:
- `bin/i18n/crowdin/codeorg_etags.json`, 7250 changes (3625 additions, 3625 deletions).
- `dashboard/config/locales/activity_sections.ar-SA.json`, 1089 changes (922 additions, 167 deletions).
- `dashboard/config/locales/scripts.pt-BR.yml`, 576 changes (571 additions, 5 deletions).
- `dashboard/config/locales/dynamic_instructions.pt-BR.json`, 459 changes (459 additions, 0 deletions).
- `dashboard/config/locales/scripts.uk-UA.yml`, 350 changes (350 additions, 0 deletions).

Top 5/83 commits sorted by the number of changes:
- Commit: hoc i18n updates
  - 20078 changes (20002 additions, 76 deletions).
  - 160 files changed.
  - File with the most changes: `i18n/locales/fil-PH/hourofcode/ph.yml`, 129 changes (127 additions, 2 deletions).
- Commit: etags updates
  - 7440 changes (3720 additions, 3720 deletions).
  - 3 files changed.
  - File with the most changes: `bin/i18n/crowdin/codeorg_etags.json`, 7250 changes (3625 additions, 3625 deletions).
- Commit: dashboard i18n updates - Portuguese, Brazilian
  - 2502 changes (2169 additions, 333 deletions).
  - 25 files changed.
  - File with the most changes: `dashboard/config/locales/scripts.pt-BR.yml`, 576 changes (571 additions, 5 deletions).
- Commit: dashboard i18n updates - Arabic
  - 1664 changes (1452 additions, 212 deletions).
  - 15 files changed.
  - File with the most changes: `dashboard/config/locales/activity_sections.ar-SA.json`, 1089 changes (922 additions, 167 deletions).
- Commit: apps i18n updates
  - 894 changes (676 additions, 218 deletions).
  - 148 files changed.
  - File with the most changes: `apps/i18n/applab/fa_ir.json`, 158 changes (79 additions, 79 deletions).

**Review by wilkie — APPROVED** [review]
I'm only sampling diffs from hoc, and not seeing anything out of the ordinary. Looks more or less alright.

**Review by mgc1194 — APPROVED** [review]
fixed a couple nits, the rests looks good.

**Pull request merged**